resources/init-container.sh

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env bash
 
-echo "---- Generating CDN ---"
-rm -rf cdn/deploy
-cd cdn
-php -d include_path=/var/www/html/_includes:. cdnrefresh.php
-cd ..
+if [[ ! $1 =~ "debug"]]; then
```

Comment:
I think the bash needs whitespace before the closing bracket
```suggestion
if [[ ! $1 =~ "debug" ]]; then
```